### DISCUSSION TITLE
Backup ADIF with cron

### DISCUSSION THREAD

**Discussion by devinberg** [magicbug/Cloudlog#1836] [Q&A]
I've tried to setup a cronjob to back-up my ADIF file using: `curl --silent https://log.n9drb.com/index.php/backup/adif &>/dev/null` and then push the resulting file to another location. However, when I run this, I only get the header info in the ADIF file and not any of the QSO information. I believe this is probably because I'm accessing that page while not logged in. Is it possible to allow that script to run as a cronjob?

**Comment by rayslinky** [comment]
Check out #1646
